service/local-dev/run_server.sh

```diff
@@ -21,7 +21,7 @@ while getopts ":av" arg; do
 done
 
 export TANAGRA_DATABASE_NAME=tanagra_db
-export TANAGRA_DB_INITIALIZE_ON_START=false
+export TANAGRA_DB_INITIALIZE_ON_START=true
```

Comment:
This is helpful for testing, but please leave it set to `false` in the main branch. This is the setting we'll deploy with, so I'd like to keep it as the testing default as well.

Reply:
done!